documentation/src/mkdocs.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+site_name: 'Exploring Pravega Flink Connectors'
+site_url: 'http://pravega.io/'
+site_description: 'Documentation to help you get familiar with Pravega Flink Connectors'
+
+repo_url: 'https://github.com/pravega/flink-connectors'
+edit_uri: 'tree/master/documentation/src/docs'
+
+theme: 'material'
+
+# Extensions
+markdown_extensions:
+  - admonition
+  - footnotes
+  - meta
+  - pymdownx.caret
+  - pymdownx.inlinehilite
+  - pymdownx.magiclink
+  - pymdownx.smartsymbols
+  - pymdownx.superfences
+
+nav:
+  - Overview: 'index.md'
+  - Getting Started: 'getting-started.md'
+  - Quick Start: 'quickstart.md'
+  - Features:
+      - 'Streaming': 'streaming.md'
+      - 'Batch': 'batch.md'
+      - 'Table API': 'table-api.md'
+  - Metrics: 'metrics.md'
+  - Configurations: 'configurations.md'
+  - Serialization: 'serialization.md'
+  - Releasing Connectors:
```

Comment:
Based on previous comments, I suspect we will be removing this section.

Reply:
Yes